src/containers/login/index.tsx

```diff
@@ -3,32 +3,34 @@ import { Link } from 'react-router-dom';
 import LoginForm, { LoginFormData } from '../../components/login';
 import { SupabaseClient } from '@supabase/supabase-js';
 import { useNavigate } from 'react-router-dom';
+import { LoginPageContainer } from './index.styles';
 
 interface LoginContainerProps {
   supabase: SupabaseClient;
 }
 
-export const LoginContainer: React.FC<LoginContainerProps> = ({supabase}) => {
-
+export const LoginContainer: React.FC<LoginContainerProps> = ({ supabase }) => {
   const navigate = useNavigate();
 
   async function handleLoginFormSubmit(data: LoginFormData) {
-    const {error} = await supabase.auth.signInWithPassword({
+    const { error } = await supabase.auth.signInWithPassword({
       email: data.email,
-      password: data.password
+      password: data.password,
     });
 
     if (error)
       console.warn('Encountered an error when logging in.', error.cause);
-    else
-      return navigate("/overview")
-    };
+    else return navigate('/overview');
+  }
 
   return (
-    <div>
+    <LoginPageContainer>
       <h2>Login</h2>
       <LoginForm onSubmit={handleLoginFormSubmit} />
-      <p>Not registered yet? Register for an account <Link to='/register'>here</Link>.</p>
-    </div>
+      <p>
+        Not registered yet? Register for an account{' '}
```

Comment:
What does the `{' '}` do?

Reply:
that's a leftover and forgot to remove - it's auto generated and is basically a single space character into the rendered HTML

Reply:
Ah right - so if we don't have that, then the space in between the text and the `<p>` would be ignored? Or can we remove it?